recipes/jthread-lite/all/test_package/conanfile.py

```diff
@@ -0,0 +1,22 @@
+import os
+from conans import ConanFile, CMake, tools
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake", "cmake_find_package_multi"
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def package_info(self):
+        if not self.settings.os == "Windows":
+            self.cpp_info.cxxflags = ["-pthread"]
+            self.cpp_info.system_libs.extend(["pthread"])
```

Comment:
This should be apart of the recipe not the test_package

Reply:
Thank you @prince-chrismc , my brain tries hard to not let me become knowledgeable about conan ;)